### DISCUSSION TITLE
Welcome to Team-56_QR-GEN-Frontend Discussions!

### DISCUSSION THREAD

**Discussion by Gemenebe** [zuri-training/Team-56_QR-GEN-Frontend#14] [Announcements]
Well-done guys may the odds be to our favour 
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Koboj** [comment]
Hi everyone

My name is Kunle-Ola Olayiwola.

I'm a product designer in team 56 and we are building a user friendly QR generating website with the collaboration of our developers (Frontend and Backend)

**Reply by E-MARIO2022** [reply]
I am Emmanuel Ejianreh, a Product Designer.
It's been exciting working on the project
